### PR TITLE
Support comma list of plugins for --disable, --enable options.

### PR DESCRIPTION
**Pull request by jmwebservices**
Curly brace expansion does not work in Windows and I needed to disable multiple plugins using a single option.

This PR makes it possible to enable/disable multiple plugins by specifying a comma list of plugin names. Previous means of setting `--disable` and `--enable` are unaffected. `changePluginsState()` remains responsible for validating plugin names.

### CODE REVIEW

**Comment by DeeDeeG** [comment]
Hi all,

I would be strongly for this, if only because of how much it improves documentation for [enabling/disabling] multiple plugins.

My experience with disabling plugins:

I just tried using SVGO from the command-line for the first time today. (Well, I had used it before, but without supplying any arguments. The defaults are good.)

I dug into the various options, and it was totally unclear how to disable ALL the plugins at once, so I could progressively enable only the ones I want and see what impact they have. I had to search the web for this info. I did find the info, but the formatting they used there lead me to try it with commas _and spaces_ first (`--disable={PLUGIN1, PLUGIN2, PLUGIN3}`) which causes errors. I realized you need no spaces.

Long story short, the example this PR adds in README.md, of how to enable or disable multiple plugins at once, is very valuable.

I don't have as strong an opinion about brackets vs. no brackets, since I can live with the brackets, and in fact have just gotten through learning to use them, but I admit the "without the brackets" (`--disable=PLUGIN1,PLUGIN2,PLUGIN3`) format would probably seem more natural to a fair amount of people.

If @jmwebservices is really correct about this enabling control of multiple plugins on Windows, that would be a show-stopper for using this on Windows for me. I was thinking of doing some of this on Windows soon, now I know I might not be able to without this PR. Big thumbs up from me for this PR.

**Comment by DeeDeeG** [comment]
Hmm, so the README now adequately explains the bracket method:

```
  --disable=PLUGIN : Disable plugin by name, "--disable={PLUGIN1,PLUGIN2}" for multiple plugins
  --enable=PLUGIN : Enable plugin by name, "--enable={PLUGIN3,PLUGIN4}" for multiple plugins
```

However, I'd like to point out (hope this isn't off-topic) the --help message _does NOT_ explain the bracket method. I'd like to request the --help message be updated, else I'll open a PR for that if I can figure out how to do so.

**Comment by jmwebservices** [comment]
@DeeDeeG Great recommendations. Feel free to fork my repo and make a PR to the main repo with your changes. SVGO's repo owner has made no attempt to consider my PR (and others) so I do not plan on investing more time. For my projects, I am using my fork.

**Comment by GreLI** [comment]
SVGO v1.2.0 has been just released!

**Comment by DeeDeeG** [comment]
@GreLI Thank you!
